cmd/server/main.go

```diff
@@ -15,51 +15,104 @@ import (
 )
 
 const (
-	ocidRoot   = "/var/lib/ocid"
-	conmonPath = "/usr/libexec/ocid/conmon"
-	pausePath  = "/usr/libexec/ocid/pause"
+	ocidRoot      = "/var/lib/ocid"
+	conmonPath    = "/usr/libexec/ocid/conmon"
+	pausePath     = "/usr/libexec/ocid/pause"
+	ociConfigPath = "/etc/ocid.conf"
 )
 
+func defaultConfig() *server.Config {
+	return &server.Config{
+		Root:         ocidRoot,
+		Runtime:      "/usr/bin/runc",
+		SandboxDir:   filepath.Join(ocidRoot, "sandboxes"),
+		ContainerDir: filepath.Join(ocidRoot, "containers"),
+		ImageStore:   filepath.Join(ocidRoot, "store"),
+		Listen:       "/var/run/ocid.sock",
+		Conmon:       conmonPath,
+		Pause:        pausePath,
+		SELinux:      false,
```

Comment:
Not crazy aout this, I would change this to 

SELinux: selinux.SelinuxEnabled()